spec/classes/foreman_proxy__plugin__container_gateway_spec.rb

```diff
@@ -13,16 +13,46 @@
             '---',
             ':enabled: https',
             ":pulp_endpoint: https://#{facts[:fqdn]}",
-            ':sqlite_db_path: /var/lib/foreman-proxy/smart_proxy_container_gateway.db'
+            ':sqlite_db_path: /var/lib/foreman-proxy/smart_proxy_container_gateway.db',
+            ':db_connection_string: postgres://foreman-proxy:@:/container_gateway'
```

Comment:
isn't this missing a password?

Reply:
Yep! It's using unix socket auth.

Reply:
Just removed the username though, that was wrong. Should have no username or password. Relies on using the foreman-proxy user.

Reply:
ah, that way round. nice